tests/test_start_date.py

```diff
@@ -11,19 +11,40 @@ class LinkedinAdsStartDateTest(TestLinkedinAdsBase):
     def name():
         return "tap_tester_linkedin_ads_start_date_test"
 
+    def get_properties(self, original: bool = True):
+        """Returns the connection properties"""
+        return {
+            "start_date" : self.START_DATE,
+            "accounts": "503491473,502890776",
+        }
+
```

Comment:
Can we remove this overriden get_properties() as the start_date will be override in the `ensure_connection(`) itself?